health_check/templates/health_check/index.html

```diff
@@ -1,32 +1,40 @@
 <html>
 <head>
-  <title>System status</title>
+  <title>
+    {% block title %}
+      System status
+    {% endblock title %}
+  </title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  {% block extra_head %}
+  {% endblock extra_head %}
```

Comment:
Again, we don't need to add too many line breaks.
```suggestion
  {% block extra_head %}{% endblock extra_head %}
```